test/box/role.result

```diff
@@ -384,7 +384,7 @@ box.schema.role.revoke("role1", "read", "universe")
 box.session.su("user")
 ---
 ...
-box.space._space.index.name:get{"_space"}[3]
+box.space._space.index.name:get{"_space"}
```

Comment:
This change and the similar changes below look unnecessary.

Reply:
Dropped changes in `test/box/role.*`, `test/engine/ddl.*` and `test/sql/ddl.*` files. There were necessary for the first version of the patch for `console.lua`.